### PR TITLE
Document the new cli for courier.

### PR DESCRIPTION
**Pull request by saeta**
Document:
1. How to download and install the cli
2. How to use it in a sample toy project.
3. How to use it in slightly more complicated projects.

### CODE REVIEW

**Comment by saeta** [comment]
PTAL @eboto ? Does this cover it appropriately? Do the download & install instructions work for you? Any other feedback or comments?

File: gettingstarted/index.md [hunk 1]
```diff
@@ -5,6 +5,67 @@ layout: page
...
+
+record Fortune {
+  message: string
+}
+~~~
+
+Then, in the the root of the project (`myProject`), to generate typescript bindings, execute:
+
+~~~
+$ courier ts output /tmp src
```
Comment (eboto) [inline gettingstarted/index.md]: Given the project structure `myProject/schemas/org/example/fortune/Fortune.courier`, this line should probably be `courier ts output /tmp schemas`

File: gettingstarted/index.md [hunk 2]
```diff
@@ -5,6 +5,67 @@ layout: page
...
+### Command line generator
+
+The easiest way to begin using courier is via the standalone command line tool. The standalone tool
+generates type safe bindings for any courier-supported language. Assuming Java (JRE) is already
+installed on your system, you can download the binary from
+[Bintray](https://dl.bintray.com/coursera/generic/courier) and install it by doing the following:
+
+~~~sh
+cd /tmp && \
+wget https://dl.bintray.com/coursera/generic/courier && \
```
Comment (eboto) [inline gettingstarted/index.md]: Oops my machine (mac, el cap) didn't come with wget...I had to use `curl -L https://dl.bintray.com/coursera/generic/courier >> courier && \`

**Comment by eboto** [comment]
This is a great addition! I encountered only the two problems listed in-line. Not only that, but I learned a bit about how common schemas are meant to be integrated. Bravo!

**Comment by eboto** [comment]
Not sure what's up with the travis build though. Appears to be unrelated to the changes.

File: gettingstarted/index.md (hunk 1 above)
Comment (saeta) [inline gettingstarted/index.md]: You are absolutely right. Fixing...

**Comment by saeta** [comment]
Hey @eboto thanks for the really helpful review. I think `curl` is more likely to be available across platforms, so I've switched the instructions to use that. Also, thank you for catching the gaffe in the sample invocation. I'm going to merge this in so that it's available for everyone.

Thank you very much for your great work on `courier`! Let me know if this download & install method doesn't meet your needs.

**Comment by saeta** [comment]
(Oh, and the travis build was just confused, because the `.travis.yml` file wasn't there, so it guessed it was a ruby project, but then got all confused. I've tweaked the travis-ci settings such that it won't attempt to build unless the `.travis.yml` file is present, so this shouldn't be an issue going forward.)

**Comment by eboto** [comment]
This is super rad, Brennan. My team will start using the cli binaries on our ts projects this week I bet =)

**Comment by saeta** [comment]
I'm very glad to hear that! Let me know how the adoption goes & what issues you encounter. :-)

**Pull request merged**